modules/lm-coursier/src/main/scala/lmcoursier/CoursierConfiguration.scala

```diff
@@ -53,9 +53,7 @@ import scala.concurrent.duration.Duration
   @since
   missingOk: Boolean = false,
   @since
-  sbtClassifiers: Boolean = false,
-  @since
-  providedInCompile: Boolean = true
+  sbtClassifiers: Boolean = false
```

Comment:
Let's keep this one not break binary compatibility, just in case.

```suggestion
  sbtClassifiers: Boolean = false,
  @since
  providedInCompile: Boolean = true // unused, kept for binary compatibility
```

Reply:
ok. I'll add a commit for this but it should be `false`

Reply:
Addressed in https://github.com/coursier/sbt-coursier/pull/262/commits/3d05a51df14f0943a05667861ac4314ee65a42a4